test/test_nestedtensor.py

```diff
@@ -1053,6 +1053,7 @@ def test_embedding(self, device, layout):
         )
         emb = torch.nn.Embedding(100, 8, device=device)
         y = emb(x)
+        y.sum().backward()
```

Comment:
Is this good enough? Should I add a check for the gradients?

Reply:
Hey @schmidt-jake - the testing comment I left involving NJT + OpInfo entries should give you gradient checking automatically! lmk if you have questions